catroid/src/main/java/org/catrobat/catroid/content/bricks/RepeatUntilBrick.java

```diff
@@ -38,8 +38,7 @@
 
 public class RepeatUntilBrick extends FormulaBrick implements CompositeBrick {
 
-	private transient EndBrick endBrick = new EndBrick(this);
-
+	private EndBrick endBrick = new EndBrick(this);
```

Comment:
What is the reason that you deleted the *getEndBrick()* method in all classes, but not for this one?

Reply:
I reworked it to not use any xml change, this also fixes the app crashes.
I forgot to delete this getEndBrick().